ArduCopter/AP_Arming.cpp

```diff
@@ -185,7 +186,7 @@ bool AP_Arming_Copter::parameter_checks(bool display_failure)
     if ((checks_to_perform == ARMING_CHECK_ALL) || (checks_to_perform & ARMING_CHECK_PARAMETERS)) {
 
         // ensure ch7 and ch8 have different functions
```

Comment:
check that channels have different functions

Reply:
Fixed